bitmovin.php

```diff
@@ -204,25 +218,47 @@ function getPlayerTable($id)
     $player_version_url = get_post_meta($id, "_config_player_version_url", true);
     $player_key = get_post_meta($id, "_config_player_key", true);
 
+    $analytics_enabled = json_decode(get_post_meta($id, "_analytics_enabled", true));
+
     $playerTable = '<table class="wp-list-table widefat fixed striped">';
 
     $playerTable .= getTableRowSelect("Channel", "config_player_channel", $player_channel, $playerChannels);
     $playerTable .= getTableRowSelect("Version", "config_player_version", $player_version, array());
     $playerTable .= getTableRowInput("Version Url", "config_player_version_url", $player_version_url);
     $playerTable .= getTableRowSelect("Player Key", "config_player_key", $player_key, array());
     $playerTable .= "</table>";
+    $playerTable .= '<br/><input onClick="handleAnalyticsCheckboxChange(this)"' . ($analytics_enabled == 'on' ? 'checked="checked"' : '') . ' type="checkbox" id="analytics_enabled" name="analytics_enabled">';
+    $playerTable .= '<label for="analytics_enabled">Enable Analytics</label>';
 
     return $playerTable;
 }
 
+function getAnalyticsTable($id)
+{
+    $analytics_key = get_post_meta($id, "_config_analytics_key", true);
+    $analytics_videoid = get_post_meta($id, "_config_analytics_videoid", true);
+    $analytics_customConf = json_decode(get_post_meta($id, "_config_analytics_custom_conf", true));
+
+    $analyticsTable = '<table class="wp-list-table widefat fixed striped">';
+
+    $analyticsTable .= getTableRowSelect("Analytics Key", "config_analytics_key", $analytics_key, array());
+    $analyticsTable .= getTableRowInput("Video ID", "config_analytics_videoid", $analytics_videoid, "Video id");
+    $analyticsTable .= "<tr><td>Custom configuration</td><td><pre>var analyticsConf = {<br><div class='intend1'><textarea id='config_analytics_custom' name='config_analytics_custom_conf' placeholder='Insert your custom configuration here, e.g.:\n title: \"A descriptive video title\"\n'>" . $analytics_customConf . "</textarea></div>};</pre></td></tr>";
+    $analyticsTable .= "<tr><td colspan='2' class='hint'>An Overview about all available configuration options can be found in our <a href='https://developer.bitmovin.com/hc/en-us/articles/115001689833' target='_blank'>documentation</a>. <br/>HINT: The configuration properties set above the custom configuration will override conflicts in this configuration.</td></tr>";

```

Comment:
pls remove that `An Overview about all available configuration...` sentence as it's for player